sample/FOSSASIA15/events

```diff
@@ -7,12 +7,12 @@
     "start_time": "2015-03-12T14:00:00", 
     "end_time": "2015-03-15T17:00:00", 
     "timezone": "Asia/Singapore", 
-    "description": "FOSSASIA is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.", 
+    "description": "The Smart City with Open Source, Big Data, Design Thinking and Free Knowledge Tools at Asia's Premier 2015 Open Technology Event", 
+    "social_links", 
     "background_image": null, 
     "logo": "img/fossasia-long.png", 
     "organizer_name": "FOSSASIA", 
-    "organizer_description": "The Smart City with Open Source, Big Data, Design Thinking and Free Knowledge Tools at Asia's Premier 2015 Open Technology Event", 
-    "social_links": [
+    "organizer_description": "FOSSASIA is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.": [
         {

```

Comment:
@mahikaw I think you forgot to put "social_links" tag here after "organizer_description". Starting array directly?